gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/ImportCRDUseCase.java

```diff
@@ -286,7 +286,7 @@ private ApiCRDStatus update(Input input, Api existingApi) {
                         return Map.entry(
                             key,
                             updatePlanDomainService
-                                .update(initPlanFromCRD(plan), plan.getFlows(), existingPlanStatuses, api, input.auditInfo)
+                                .update(initPlanFromCRD(plan), plan.getFlows(), api, input.auditInfo, existingPlanStatuses)
```

Comment:
Could you reorder the parameter to keep original order to prevent updating existing code

Reply:
done